components/list/ListItem.js

```diff
@@ -80,7 +86,14 @@ const factory = (ripple, ListItemLayout, ListItemContent) => {
       const children = this.groupChildren();
       const content = <ListItemLayout theme={theme} {...children} {...other} />;
       return (
-        <li className={`${theme.listItem} ${className}`} onClick={this.handleClick} onMouseDown={onMouseDown} onTouchStart={onTouchStart}>
+        <li
+          className={`${theme.listItem} ${className}`}
+          onClick={this.handleClick}
+          onMouseDown={onMouseDown}
+          onTouchStart={onTouchStart}
+          onKeyDown={this.handleEnter}
+          tabIndex={0}
+        >
           {to ? <a href={this.props.to}>{content}</a> : content}
           {children.ignored}
         </li>
```

Comment:
the handleEnter should account for when there is a `to` prop added. The `<a>` it creates, receives keyboard focus, so an user would have to tab twice to get in. If we are removing the tabIndex from the `<a>`, we need to separately address how screen reader users will be aware of the link action.